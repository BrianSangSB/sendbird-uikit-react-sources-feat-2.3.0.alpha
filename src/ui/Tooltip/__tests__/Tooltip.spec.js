import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Tooltip from "../index";

describe('Tooltip', () => {
  it('should have className', function () {
    const className = "example-text";
    const text = 'Sravan S, Hoon Baek, Chong Bu, Mickey, Leo Shin, Doo Rim, Tez Park, Harry Kim, and you';
    const component = shallow(
      <Tooltip className={className}>
        {text}
      </Tooltip>
    );

    expect(
      component.find(".sendbird-tooltip").hasClass(className)
    ).toBe(true);
  });

  it('should do a snapshot test of the ReactedUsers DOM', function () {
    const className = "example-text";
    const text = 'Sravan S, Hoon Baek, Chong Bu, Mickey, Leo Shin, Doo Rim, Tez Park, Harry Kim, and you';
    const component = renderer.create(
      <Tooltip className={className}>
        {text}
      </Tooltip>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
