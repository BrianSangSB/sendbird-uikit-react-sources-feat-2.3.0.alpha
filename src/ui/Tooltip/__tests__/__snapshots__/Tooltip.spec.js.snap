// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip should do a snapshot test of the ReactedUsers DOM 1`] = `
<div
  className="example-text sendbird-tooltip"
>
  <span
    className="sendbird-tooltip__text sendbird-label sendbird-label--caption-2 sendbird-label--color-oncontent-1"
  >
    Sravan S, Hoon Baek, Chong Bu, Mickey, Leo Shin, Doo Rim, Tez Park, Harry Kim, and you
  </span>
</div>
`;
